server/test/unit/user.test.js

```diff
@@ -226,6 +228,13 @@ describe("DELETE /users/:id", () => {
         expect(res.statusCode).toEqual(200);
     })
 
+    it("fails if the user to be deleted does not exist", async () => {
+        let res = await request(app)
+            .delete('/users/' + faker.database.mongodbObjectId())
+            .set('Accept', 'application/json')
+        expect(res.statusCode).toEqual(400);
```

Comment:
400 = Bad Request
If the user doesn't exist, we would typically use 404 to indicate "No User Found"
@benhsm

Reply:
Ah, good catch, thanks!